front/app/api/graph_data_units/requestTypes.ts

```diff
@@ -151,7 +151,11 @@ interface ParticipationParams extends BaseParams {
   props: ParticipationProps;
 }
 
-export interface ProjectsProps extends DateProps {}
+export type ProjectReportsPublicationStatus = 'published' | 'archived';
+
+export interface ProjectsProps extends DateProps {
+  publication_statuses: ProjectReportsPublicationStatus;
```

Comment:
It says 'statuses', but it's only a string with a single status. Shouldn't it be an array of statuses?

Reply:
Wasn't the issue also that they want both published and archived in the same widget?

Reply:
> Wasn't the issue also that they want both published and archived in the same widget?

Yes, that was the plan.